Support decimal point in estimation field

```
panic: json: cannot unmarshal number 0.5 into Go struct field Estimate.issues.estimate.value of type int

goroutine 1 [running]:
main.main()
	/github/workspace/main.go:118 +0x285
```

--- a/gzc/gzc.go
+++ b/gzc/gzc.go
@@ -59,7 +59,7 @@ func (c *Client) createURL(resource string) string {
 }
 
 type Estimate struct {
-	Value int `json:"value"`
+	Value float64 `json:"value"`
 }
 type Pipeline struct {
 	WorkspaceID string `json:"workspace_id"`

--- a/zed.go
+++ b/zed.go
@@ -3,7 +3,6 @@ package main
 import (
 	"context"
 	"fmt"
-	"strconv"
 
 	"github.com/google/go-github/v42/github"
 	"github.com/seriar-org/zed/gdm"
@@ -24,7 +23,7 @@ type Node struct {
 	Repo        string
 	Owner       string
 	IssueID     int
-	StoryPoints int
+	StoryPoints float64
 	State       string
 	Title       string
 	Assignee    string
@@ -35,7 +34,7 @@ type Node struct {
 func (n *Node) MermaidNodeText() string {
 	sp := "?"
 	if n.StoryPoints > 0 {
-		sp = strconv.Itoa(n.StoryPoints)
+		sp = fmt.Sprintf("%.1f", n.StoryPoints)
 	}
 	text := fmt.Sprintf("%s/%s#%d<br/>%s<br/>%s SP<br/>%s<br/>%s", n.Owner, n.Repo, n.IssueID, n.Title, sp, n.Assignee, n.State)
 	if n.IsExternal {

